internal/cmd/get/get_test.go

```diff
@@ -71,3 +74,62 @@ func Test_initialiseProjectInCurrentDir(t *testing.T) {
 		})
 	}
 }
+
+func Test_run(t *testing.T) {
+	type args struct {
+		ctx  context.CLIContext
+		opts *options
+		cmd  *cobra.Command
+		args []string
+	}
+	ctx := context.CLIContext{
+		Config: context.CLIConfig{},
+		Paths:  context.Paths{
+			ProjectConfigFile: path.Join(os.TempDir(), "config.yaml"),
+			ProjectInstallDir: os.TempDir(),
+			GlobalInstallDir:  os.TempDir(),
+		},
+	}
+	rootCmdWithGet := cobra.Command{}
+	emptyCmd := cobra.Command{}
+
+	getCmd := NewCommand(ctx)
+	rootCmdWithGet.AddCommand(getCmd)
+
+	tests := []struct {
+		name  string
+		args  args
+		error error
+	}{
+		{
+			name: "should return error 'Cannot get already registered command 'get''",
```

Comment:
the error message isn't clear for me. How about changing it to `The command 'get' is already registered`?